src/components/TransferPreview/TransferPreview.tsx

```diff
@@ -53,7 +64,7 @@ const TransferPreview = () => {
 		<div
 			className={`${styles.TransferPreview} border-primary border-rounded blur`}
 		>
```

Comment:
Please check if it works `border-rounded blur` on firefox. In my experience it did not work on firefox.